Document/0x06b-Basic-Security-Testing.md

```diff
@@ -327,15 +327,40 @@ Keychain Data: WOg1DfuH
 
 Note however that this binary is signed with a self-signed certificate with a "wildcard" entitlement, granting access to *all* items in the Keychain - if you are paranoid, or have highly sensitive private data on your test device, you might want to build the tool from source and manually sign the appropriate entitlements into your build - instructions for doing this are available in the GitHub repository.
 
-<!--
-
 ##### Security Profiling with Introspy
 
-Intospy is an open-source security profiler for iOS released by iSecPartners. Built on top of substrate, it can be used to log security-sensitive API calls on a jailbroken device.  The recorded API calls sent to the console and written to a database file, which can then be converted into an HTML report using Introspy-Analyzer <code>[32]</code>.
+Introspy is an open-source security profiler for iOS released by iSecPartners. Built on top of Substrate, it can be used to log security-sensitive API calls on a jailbroken device. The recorded API calls sent to the console and written to a database file, which can then be converted into an HTML report using Introspy-Analyzer. 
```

Comment:
Can you please add links to Introspy and Introspy-Analyzer (Github repos). Please use Inline references as descbribed here https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md